build/template-Build-run-tests-sign.yml

```diff
@@ -28,7 +28,7 @@ steps:
 - task: UseDotNet@2
   displayName: 'Use .Net Core SDK 9.x'
   inputs:
-    version: 9.0.100-preview.4.24267.66
+    version: 9.0.100-preview.7.24407.12
```

Comment:
we should move to RC1 https://dotnet.microsoft.com/en-us/download/dotnet/9.0

Reply:
Updated to rc1.